xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/DisplayerDate.vue

```diff
@@ -0,0 +1,167 @@
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+
+<!--
+  DisplayerDate is a custom displayer that displays the entry value as dates
+-->
+<template>
+  <!--
+    Uses the BaseDisplayer as root element, as it handles for us
+    all the displayer default behavior
+  -->
+  <BaseDisplayer
+    ref="baseDisplayer"
+    class="displayer-date"
+    :property-id="propertyId"
+    :entry="entry"
+  >
+
+    <!-- Provide the Date Viewer widget to the `viewer` slot -->
+    <template #viewer>
+        <div>{{ valueFormatted }}</div>
+    </template>
+
+    <!-- Provide the Date Editor widget to the `editor` slot -->
+    <!-- TODO: use daterangepicker widget -->
+    <template #editor>
+      <!-- A simple text input that will be upgraded to have a date picker -->
+      <input
+        class="editor-date"
+        ref="editorDate"
+        type="text"
+        size="1"
+        :value="valueFormatted"
+        v-on-inserted="upgradeDatePicker"
+        v-autofocus
+      />
+    </template>
+
+  </BaseDisplayer>
+</template>
+
+
+<script>
+
+import displayerMixin from "./displayerMixin.js";
+import BaseDisplayer from "./BaseDisplayer.vue";
+import daterangepicker from "daterangepicker";
+import moment from "moment";
+import $ from "jquery";
+
+export default {
+
+  name: "displayer-date",
+
+  components: {
+    BaseDisplayer,
+  },
+
+  // Add the displayerMixin to get access to all the displayers methods and computed properties inside this component
+  mixins: [displayerMixin],
+
+
+  computed: {
+    format () {
+      return "YYYY/MM/DD HH:mm";
+    },
+    // Date formatted to be human-readable
+    valueFormatted () {
+      return moment(+this.value).format(this.format);
+    },
+
+    editorConfig () {
+      return {
+        drops: 'down',
+        opens: 'right',
+        autoUpdateInput: true,
+        autoApply: true,
+        singleDatePicker: true,
+        showDropdowns: true,
+        timePicker: true,
+        timePicker24Hour: true,
+        alwaysShowCalendars: true,
+        locale: {
+          format: this.format,
+        },
```

Comment:
It's more complex than this. The date picker configuration needs to follow the date format. For instance, if the date format doesn't include the time then the picker should disable the time component. We did this for our [``dateTimePicker.js``](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/dateTimePicker.js) which you should use instead.